tests/publiccloud/run_ltp.pm

```diff
@@ -45,35 +45,41 @@ sub instance_log_args
 sub run {
     my ($self)   = @_;
     my $arch     = check_var('PUBLIC_CLOUD_ARCH', 'arm64') ? 'aarch64' : 'x86_64';
-    my $ltp_repo = get_var('LTP_REPO', 'http://download.suse.de/ibs/QA:/Head/' . generate_version("-") . '/' . $arch . '/');
+    my $ltp_repo = get_var('LTP_REPO', 'https://download.opensuse.org/repositories/benchmark:/ltp:/devel/' . generate_version("_") . '/');
```

Comment:
> .... But I understand that you will use `LTP_REPO` variable to change it.


yes exactly . in conjunction with `LTP_RPM_MANUAL_UPLOAD`  because we can not install from internal repo into public VM . so in case of IBS we need to download rpm into openQA VM and than upload it into Public Cloud

>  ... till it reaches beta phase .

Public Cloud developers start producing first images **after** first Beta :) so it should not be a problem